crates/nu-command/src/conversions/into/datetime.rs

```diff
@@ -67,8 +82,11 @@ impl Command for IntoDatetime {
             (Type::String, Type::Date),
             (Type::List(Box::new(Type::String)), Type::List(Box::new(Type::Date))),
             (Type::table(), Type::table()),
+            // FIXME: https://github.com/nushell/nushell/issues/15485
+            // 'record -> any' was added as a temporary workaround to avoid type inference issues.
             (Type::record(), Type::record()),
-            (Type::Nothing, Type::table()),
+            (Type::record(), Type::Date),
+            (Type::record(), Type::Any),
```

Comment:
If I understood @132ikl  correctly the Any arm needs to appear as the first one so it is picked with the more specific documentation versions below.

Reply:
Right, I forgot about that.
Fixed in the last commit